packages/help-center/src/hooks/use-chat-status.ts

```diff
@@ -1,14 +1,19 @@
-import { isTestModeEnvironment } from '@automattic/zendesk-client';
+import { isTestModeEnvironment, useCanConnectToZendeskMessaging } from '@automattic/zendesk-client';
 import { useSupportActivity } from '../data/use-support-activity';
 import { useSupportStatus } from '../data/use-support-status';
 
 export default function useChatStatus() {
-	const { data: supportStatus } = useSupportStatus();
+	const { data: canConnectToZendesk } = useCanConnectToZendeskMessaging();
```

Comment:
Also confused by the fact that `canConnectToZendesk` is returned by `useHelpCenterContext`. Is it the same value?

Reply:
Yes, that's a mistake in the design that I didn't want to refactor. It's a common mistake where people pass down values that are globally accessible via hooks.